### PR TITLE
HomeWork on DataTypes is done

### PR DESCRIPTION
**Pull request by hannanezrivnanna**

### CODE REVIEW

File: Exercises/2-by-value.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 'use strict';
 
-const inc = null;
+const inc = number => number * number;
```
Comment (tshemsedinov) [inline Exercises/2-by-value.js]:
```suggestion
const inc = x => ++x;
```

File: Exercises/3-by-reference.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 'use strict';
 
-const inc = null;
+const inc = Num => Num.n = 0;
```
Comment (tshemsedinov) [inline Exercises/3-by-reference.js]:
```suggestion
const inc = num => {
  num.n++;
};
```

File: Exercises/4-count-types.js [hunk 3]
```diff
@@ -1,5 +1,14 @@
 'use strict';
 
-const countTypesInArray = null;
+const countTypesInArray = () => {
+  const array = [true, 'hello', 5, 12, -200, false, false,
```
Comment (tshemsedinov) [inline Exercises/4-count-types.js]: Remove example array from function, see other pull requests

Comment (hannanezrivnanna) [reply Exercises/4-count-types.js]: Thanks. Fixed!

**Comment by hannanezrivnanna** [comment]
Fixed after code review

**Pull request closed without merging**